agent/app/agent.go

```diff
@@ -435,6 +435,17 @@ func (agent *ecsAgent) getEC2InstanceID() string {
 	return instanceID
 }
 
+// getoutpostARN gets the Outpost ID from the metadata service
```

Comment:
> getoutpostARN gets the Outpost ID

Is it outpost id? outpost arn? aws id (from your commit message)?  Please make everything match.

Reply:
Fixed.